code/modules/scripting/Scanner/Scanner.dm

```diff
@@ -26,7 +26,7 @@
 	Loads source code.
 */
 /datum/n_Scanner/proc/LoadCode(var/c)
-	code=strict_ascii(c)
+	code = c
```

Comment:
I remember this was to fix a crash, #16026
Are we sure this doesn't happen anymore now?
iirc you could reproduce it by trying to replace a string with NTSL, like "spanish" to "español"

Reply:
Given that BYOND actually supports Unicode now, that would be pretty stunning if it still happened, but I can test it tomorrow I guess

Reply:
It doesn't crash the server, but it also doesn't render properly, so I'm glad I tested it

Reply:
Fixed the easy way by just changing all the code scanner procs to use the relevant `_char` versions of the text procs
It might be faster to not do that and instead implement the byte counting ourselves (since there's a fair bit of redundancy in it this way) but I don't really want to do that

Incidentally, my first attempt at this introduced a server crash much like the one you were concerned about, but only with invalid syntax